tests/test-constants.js

```diff
@@ -78,12 +78,12 @@ export const TestAccountCouncilMemberURI = fromEnv(
 export const MinGasPrice = fromEnv('MinGasPrice', DefaultMinGasPrice);
 export const MaxGas = fromEnv('MaxGas', DefaultMaxGas);
 export const TestSchemes = fromEnv('TestSchemes', DefaultTestSchemes);
-export const DisableStatusListTests = boolEnv(
-  fromEnv('DisableStatusListTests', 'false'),
-);
 export const DisableDidKeyAndTrustRegistryTests = boolEnv(
   fromEnv('DisableDidKeyAndTrustRegistryTests', 'false'),
 );
+export const DisableNewAccumulatorTests = boolEnv(
```

Comment:
Maybe it's better to call this variable `DisableKBAccumulatorTests`?

Reply:
Its for the keyless VB accum. as well.